pkg/reconciler/app/resources/deployment_test.go

```diff
@@ -653,13 +653,13 @@ func Test_makePodSpec(t *testing.T) {
 							Command:                  []string{"/bin/sh"},
 							Args: []string{
 								"-c",
-								"mapfs -uid 2000 -gid 2000 /nfs/volume1 /.kfmounts/nfs/volume1 & /lifecycle/entrypoint.bash -jar my-library.jar -timeout=10",
+								"mapfs -uid 2000 -gid 2000 /nfs/volume1 /.kfmounts/nfs/volume1 & exec /lifecycle/entrypoint.bash -jar my-library.jar -timeout=10",
```

Comment:
nit: can you add a test to ensure the shell escape is working?

Reply:
I went ahead and updated one of the tests to include a string that needs shell escaping.